src/main/java/org/ngafid/accounts/User.java

```diff
@@ -227,6 +239,10 @@ public static User get(Connection connection, int userId, int fleetId) throws SQ
         user.fleetAccess = FleetAccess.get(connection, user.id, fleetId);
         user.fleet = Fleet.get(connection, fleetId);
 
+        //Get the email preferences for this user
+        user.userPreferencesEmails = getUserPreferencesEmails(connection, userId);
```

Comment:
Make this lazy, since we won't need the email preferences 99% of the time. Just set to null by default and the getter function can populate it.